share/github-backup-utils/ghe-rsync

```diff
@@ -14,14 +14,26 @@ set -o pipefail
 # stderr (rsync versions >= 3.x). The complex redirections are necessary to
 # filter stderr while also keeping stdout and stderr separated.
 IGNOREOUT='^(file has vanished: |rsync warning: some files vanished before they could be transferred)'
+
+# Remove --ignore-missing-args argument if using an earlier version of rsync.
+if [[ $GHE_EXTRA_RSYNC_OPTS == *"--ignore-missing-args"* ]] && ! rsync -h | grep -q '\-\-ignore-missing-args'; then
+  GHE_EXTRA_RSYNC_OPTS=${GHE_EXTRA_RSYNC_OPTS//--ignore-missing-args/}
```

Comment:
This is going to need to be extended to strip `--ignore-missing-args` from the list of arguments passed in too else we hit the failure seen in the current macOS builds.

Reply:
Good catch, I was looking for `--ignore-missing-args` in the wrong place!